code/src/Form.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Dropdown } from 'Dropdown'
+import { Radio } from 'Radio'
+import { Text } from 'Text'
+import { Submit } from 'Submit'
+
+
+
+export const Form = () => {
+ 
+    const [dropInput, fetchDropInput] = useState("")
+    const [radioInput, fetchRadioInput] = useState("")
+    const [textInput, fetchTextInput] = useState("")
+    const [section, setSection] = useState('start')
+
+    const handleDropInput = (data) => {
+        fetchDropInput(data)
+    }
+
+    const handleTextInput = (data) => {
+        fetchTextInput(data)
+    }
+
+    const handleRadioInput = (data) => {
+        fetchRadioInput(data)
+    }
+
+    return <section class="survey">
+
+        
+        <form onSubmit={(e) => { e.preventDefault() }} >
+
+            {section === 'start' && (
+                <section>
+                    <p className="question">Stay on my wing and ill take you home?</p>
+                    <Submit title="Yes!" setSection={setSection} value='callsign'/>
+                </section>
+            )}
+
+            {section === 'callsign' && (
+                <section>
+                    <Radio handleRadioInput={handleRadioInput} />
```

Comment:
You are going to use that method inside `<Radio />` component, so it should be named `onRadioInput`, like this :
```
<Radio onRadioInput={handleRadioInput} />
```

Reply:
I did not fully understand this comment but I changed all props to onRadioInput, onTextInput, onDropinput. I thought it was ok to name the prop to be the same as the parameter?